src/DatePicker/DatePicker.ios.js

```diff
@@ -71,55 +67,43 @@ export default class iOSDatePicker extends React.Component<Props, State> {
 
     return (
       <Modal
-        transparent
-        visible={isVisible}
+        isVisible={isVisible}
         onRequestClose={this.handleDismiss}
-        animationType="slide"
+        onBackdropPress={this.handleDismiss}
       >
-        <TouchableWithoutFeedback onPress={this.handleDismiss}>
-          <View style={styles.backdrop}>
-            <View style={styles.content}>
-              <DatePickerIOS
-                date={date}
-                onDateChange={this.handleChangeDate}
-                mode={mode}
-                maximumDate={maxDate}
-                minimumDate={minDate}
-              />
-              <View style={styles.buttonsContainer}>
-                <Touchable
-                  onPress={this.handleDismiss}
-                  style={styles.confirmButton}
-                >
-                  <Text style={styles.buttonText}>Cancel</Text>
-                </Touchable>
-                <Touchable
-                  style={styles.confirmButton}
-                  onPress={this.handleConfirm}
-                >
-                  <Text style={styles.buttonText}>OK</Text>
-                </Touchable>
-              </View>
-            </View>
+        <View style={styles.content}>
+          <DatePickerIOS
+            date={date}
+            onDateChange={this.handleChangeDate}
+            mode={mode}
+            maximumDate={maxDate}
+            minimumDate={minDate}
+          />
+          <View style={styles.buttonsContainer}>
+            <Touchable
+              onPress={this.handleDismiss}
+              style={styles.confirmButton}
+            >
+              <Text style={styles.buttonText}>Cancel</Text>
```

Comment:
This should probably be passed in from the consumer. Since we would want to translate it.

Reply:
One way we could achieve this is by adding additional prop:
```
type Props = {|
...
+label: {|
   +cancel: string,
   +confirm: string,
|},
...
|};
```

Reply:
But please do not restrict it to string, it could also be any component, `Translation` or `Text`

Reply:
So replace `string` with `React.Node`?

Reply:
That should be good enough yes 😊

Reply:
Updated